robot/EDA/tests/browser/contacts/verify_preferred_phone.robot

```diff
@@ -1,6 +1,7 @@
 *** Settings ***
 Documentation
 ...     Verify Preferred Phone functionality
+Force Tags      unstable
```

Comment:
@dashattuck Does #1006 supercede this? It looks like this PR might no longer be required if we're marking the tests unstable there.

Reply:
@robertbwatson - correct this PR is no longer required as #1006 supersedes it.